### PR TITLE
Wfst in celery II

### PR DESCRIPTION
**Pull request by index-git**
Part of issue #159

- ~~Tests~~
- ~~Layman Test Client~~
- ~~Changlelog~~
- ~~Documentation~~

Optionally
- Add dependency to doc/dependencies.md

### CODE REVIEW

File: src/layman/common/consts.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+REQUEST_METHOD_DELETE = 'delete'
```
Comment (jirik) [inline src/layman/common/consts.py:1]: It's little bit strange we are mixing here HTTP requests and WFS requests as they are from different network levels.

I suggest to use one set of variables for HTTP requests and another set for locks (HTTP request names + WFST).

Comment (jirik) [reply src/layman/common/consts.py:1]: Move it to common/__init__.py

**Review by jirik — CHANGES_REQUESTED** [review]

File: src/layman/common/redis.py [hunk 2]
```diff
@@ -70,22 +71,23 @@ def solve_locks(workspace, publication_type, publication_name, error_code, metho
     )
     if current_lock is None:
         return
-    if method not in ['patch', 'delete', 'wfst', ]:
+    if method not in [consts.REQUEST_METHOD_PATCH, consts.REQUEST_METHOD_DELETE, consts.REQUEST_METHOD_WFST, ]:
```
Comment (index-git) [inline src/layman/common/redis.py:74]: rename parameter `method` to `required_lock`

File: src/layman/authz/__init__.py [hunk 3]
```diff
@@ -113,7 +114,7 @@ def decorated_function(*args, **kwargs):
         actor_name = authn.get_authn_username()
         # raises exception in case of unauthorized request
         authorize(workspace, publication_type, publication_name, request.method, actor_name)
-        if workspace and publication_type and not publication_name and request.method == 'GET':
+        if workspace and publication_type and not publication_name and request.method == common.REQUEST_METHOD_GET:
```
Comment (jirik) [inline src/layman/authz/__init__.py:117]: Add `.lower()`

File: src/layman/authz/__init__.py [hunk 4]
```diff
@@ -132,7 +133,7 @@ def decorated_function(*args, **kwargs):
         if publication_type is None or workspace or publication_name:
             raise Exception(f"Authorization module is unable to authorize path {req_path}")
         actor_name = authn.get_authn_username()
-        if request.method == 'GET':
+        if request.method == common.REQUEST_METHOD_GET:
```
Comment (jirik) [inline src/layman/authz/__init__.py:136]: Add `.lower()`

**Review by jirik — CHANGES_REQUESTED** [review]

**Review by jirik — APPROVED** [review]

**Pull request merged**